File: app/src/main/java/com/lumiyaviewer/lumiya/voice/common/messages/VoiceEnableMic.java
package com.lumiyaviewer.lumiya.voice.common.messages;

import android.os.Bundle;
import com.lumiyaviewer.lumiya.voice.common.VoicePluginMessage;

public class VoiceEnableMic implements VoicePluginMessage {
    public final boolean enableMic;

    public VoiceEnableMic(Bundle bundle) {
        this.enableMic = bundle.getBoolean("enableMic");
    }

    public VoiceEnableMic(boolean z) {
        this.enableMic = z;
    }

    public Bundle toBundle() {
        Bundle bundle = new Bundle();
        bundle.putBoolean("enableMic", this.enableMic);
        return bundle;
    }
}

File: app/src/main/java/com/lumiyaviewer/lumiya/voice/common/messages/VoiceEnableMic.kt
/*
 * Decompiled with CFR 0.152.
 * 
 * Could not load the following classes:
 *  android.os.Bundle
 */
package com.lumiyaviewer.lumiya.voice.common.messages

import android.os.Bundle
import com.lumiyaviewer.lumiya.voice.common.VoicePluginMessage

class VoiceEnableMic
: VoicePluginMessage {
    Boolean enableMic

    VoiceEnableMic(Bundle bundle) {
        this.enableMic = bundle.getBoolean("enableMic")
    }

    VoiceEnableMic(Boolean bl) {
        this.enableMic = bl
    }

    @Override
    Bundle toBundle() {
        Bundle bundle = Bundle()
        bundle.putBoolean("enableMic", this.enableMic)
        return bundle
    }
}

